WebRoot/js/zimbraMail/share/model/ZmInvite.js

```diff
@@ -853,7 +855,7 @@ function(compNum) {
 ZmInvite.prototype.getLocation =
 function(compNum) {
 	var cn = compNum || 0;
-	return this.components[cn] ? this.components[cn].loc : null;
+	return this.components[cn] ? AjxStringUtil.stripTags(this.components[cn].loc) : null;
```

Comment:
As discussed, this should be done at the UI level and AjxStringUtil.htmlEncode should be used as in this case email is getting lost when its wrapped on ui with  <> .
@SatyamKumarOnline